js/src/hooks/useGoogleAdsTermsURL.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '.~/data/constants';
+
+export const ENHANCED_CONVERSION_TERMS_BASE_URL =
+	'https://ads.google.com/aw/conversions/customersettings';
+
+const useGoogleAdsEnhancedConversionTermsURL = () => {
+	return useSelect( ( select ) => {
+		const adsAccount = select( STORE_KEY ).getGoogleAdsAccount();
+
+		const url = addQueryArgs( ENHANCED_CONVERSION_TERMS_BASE_URL, {
+			ocid: adsAccount?.ocid,
```

Comment:
If the OCID is null, the link will result in a 400 error. Falling back to `0` will at least still trigger the account selection flow that will get the person to the right place.
```suggestion
			ocid: adsAccount?.ocid ?? 0,
```

Reply:
👍